source/jsonnet/northern-ireland/individual/blocks/identity-and-health/disability_other.jsonnet

```diff
@@ -90,12 +90,24 @@ local proxyTitle = {
         when: [rules.under4],
       },
     },
+    {
+      goto: {
+        section: 'End',
+        when: [rules.lastBirthdayAgeUnder(4)],
+      },
+    },
     {
       goto: {
         block: 'carer',
         when: [rules.over5],
       },
     },
+    {
+      goto: {
+        block: 'carer',
+        when: [rules.lastBirthdayAgeOver(5)],
```

Comment:
The trello card has `if 4 route to...` - what does that mean?

Reply:
Yeah, I had no idea either

Reply:
Donna has updated this in the card.